recipe/0101-Build-fix-in-a-conda-environment.patch

```diff
@@ -20,27 +20,26 @@ index 47b53a3..5526225 100755
    echo 'Using installed tensorflow.'
  else
    echo 'Installing tensorflow.'
--  pip install tensorflow==2.5.0
-+  conda install tensorflow==2.5.0
+-  pip install tensorflow==2.6.0
++  conda install -y tensorflow==2.6.0
```

Comment:
strange that this looks to be installing tensorflow twice..

Reply:
This line installs Tensorflow only if it is not installed.